src/content/docs/en/concepts/why-astro.mdx

```diff
@@ -1,45 +1,59 @@
 ---
 title: Why Astro?
-description: "Astro is an all-in-one web framework for building fast, content-focused websites. Learn more."
+description: "Astro is an all-in-one web framework for building fast, content-driven websites. Learn more."
 i18nReady: true
 ---
 
-Astro is an **all-in-one** **web framework** for building **fast,** **content-focused websites.**
+**Astro** is the web framework for building **content-driven websites** like porfolios, blogs, docs, marketing sites, landing pages, and ecommerce. 
 
-Why choose Astro over another web framework? Here are five core design principles to help explain why we built Astro, the problems that it exists to solve, and why Astro may be the best choice for your project or team.
+Astro is best known for pioneering a JavaScript frontend architecture using [islands](/en/concepts/islands/), optimized for performance, flexibility, and simplicity.
 
-#### Astro is...
+## Features
 
-1. [Content-focused](#content-focused): Astro was designed for content-rich websites.
+Astro includes everything you need built-in to get started quickly, and hundreds of integrations to extend your site. 
```

Comment:
`extend your site`  feels a bit vague to me. From my perspective, Integrations usually
- improve the Dev Experience by solving problems with enough abstraction
- improve website performance (User experience)

that being said, my suggestion would be `enhance`, `boost`, `improve`, `enrich`, `elevate`
> note: more suggestions are possible